homeassistant/components/omie/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "omie",
+  "name": "OMIE - Spain and Portugal electricity prices",
+  "codeowners": ["@luuuis"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/omie",
```

```suggestion
  "documentation": "https://www.home-assistant.io/integrations/omie",
  "integration_type": "service",
```
missing integration type, since it is not the default one (hub)